ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1602,6 +1602,23 @@ literals:
 - valve_status:
   - OPEN
   - CLOSED
+
+- failed_panel_alarm:
+  - ACTIVE
+  - INACTIVE
+- low_smoke_alarm: # for aspirating type smoke detectors
+  - ACTIVE
+  - INACTIVE
+- medium_smoke_alarm: # for aspirating type smoke detectors
+  - ACTIVE
+  - INACTIVE
+- high_zone_air_h2_concentration_alarm:
+  - ACTIVE
+  - INACTIVE
+- gas_release_status:
```

Comment:
Is this a `status` (feedback) or should it be a `command` ?

Reply:
this is status, ma'am, the BMS has no controls for all fire systems (just alarms). the fire controls are automated, split apart from the BMS.

Reply:
Is it correct to call this a `gas_release_valve_status` ?

Reply:
Yes ma'am possibly better. I'll update now